src/assets/icons/ic-file-code.svg

```diff
@@ -15,5 +15,5 @@
   -->
 
 <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M4.66667 14H13.5005C13.5661 14 13.6312 13.9871 13.6918 13.9619C13.7525 13.9368 13.8076 13.9 13.854 13.8536C13.9005 13.8071 13.9373 13.752 13.9624 13.6913C13.9876 13.6307 14.0005 13.5657 14.0005 13.5V5.5M10.5005 2H4.5C4.36739 2 4.24021 2.05268 4.14645 2.14645C4.05268 2.24021 4 2.36739 4 2.5V5.25M10.5005 2L14.0005 5.5M10.5005 2V5.5H14.0005M3.30612 7.80952L2 9.56463L3.30612 11.3197M6.02721 7.80952L7.33333 9.56463L6.02721 11.3197" stroke="#0066CC" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M4.66667 14H13.5005C13.5661 14 13.6312 13.9871 13.6918 13.9619C13.7525 13.9368 13.8076 13.9 13.854 13.8536C13.9005 13.8071 13.9373 13.752 13.9624 13.6913C13.9876 13.6307 14.0005 13.5657 14.0005 13.5V5.5M10.5005 2H4.5C4.36739 2 4.24021 2.05268 4.14645 2.14645C4.05268 2.24021 4 2.36739 4 2.5V5.25M10.5005 2L14.0005 5.5M10.5005 2V5.5H14.0005M3.30612 7.80952L2 9.56463L3.30612 11.3197M6.02721 7.80952L7.33333 9.56463L6.02721 11.3197" stroke="#3b444c" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
have we verified the impact for the color change?

Reply:
yes discussed